tests/units/clean/test_clean.py

```diff
@@ -0,0 +1,39 @@
+import os
+from datetime import datetime, timezone
+
+from airflow import DAG
+from airflow.models import DagBag
+from freezegun import freeze_time
+from pytest import fixture
+
+
+@fixture
+def dag():
+    dagbag = DagBag(dag_folder="dags/", include_examples=False)
+    assert dagbag.import_errors.get(f"dags/cleanup_logs.py") is None
+    clean_dag = dagbag.get_dag(dag_id="cleanup_logs")
+    return clean_dag
+
+
+def test_dag_loaded(dag: DAG):
+    assert dag is not None
+    assert len(dag.tasks) == 1
+
+
+@freeze_time("2023-09-20")
+@fixture
+def old_temp_dir(tmpdir, tmp_path):
+    logs_date = datetime.utcnow().astimezone(timezone.utc)
+    log_path = tmpdir.join(
+        f'logs/dag_id=test/run_id=test__{logs_date.strftime("%Y-%m-%dT%H:%M:%S.%f%z")}/task_id=test_task/attempt=1.log'
```

Comment:
here your example has `z`. Also we are testing against different `_generate_id` would it work?

Reply:
here your example has z - I made it consistent with airflow generated run_id=.. value